contrib/templates/experimental/scala-job/databricks_template_schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "properties": {
+    "project_name": {
+      "type": "string",
+      "description": "Name of project. Do not add spaces",
+      "order": 1
+    },
+    "custom_workspace_host": {
+      "type": "string",
+      "default": "{{workspace_host}}",
+      "description": "Workspace host url",
+      "order": 2
+    },
+    "artifacts_dest_path": {
+      "type": "string",
+      "description": "Destination path in Databricks where the JAR and other artifacts will be stored You can use /Workspace or /Volumes in Dedicated clusters but only /Volumes in Standard clusters.",
```

Comment:
Given that some of these questions need some extra guidance, could you use the multi-line conventions as seen in the latest templates, i.e. https://github.com/databricks/cli/blob/main/libs/template/templates/default-sql/databricks_template_schema.json#L16